requirements/test.txt

```diff
@@ -1,4 +1,7 @@
--r base.txt
+# Django and Wagtail have to installed seperate from requirements
```

Comment:
The necessity of this comment feels a little bit like a sign that we don't have these files set up optimally.

Some of our existing requirements files (`manual.txt`, `mysql.txt`, `scripts.txt`, `travis.txt`) are standalone additions to a base set of requirements necessary to work with this repository. But others (`local.txt`, `deployment.txt`, `test.txt`) include child requirements files within themselves.

How would you feel about moving `test.txt` from the latter group to the former, and directing tox to explicitly install all of the necessary child requirements?

Reply:
That's fine by me.